Gemfile

```diff
@@ -21,6 +25,8 @@ group :development do
 end
 
 group :development, :system_tests do
-  gem "beaker"
+  if RUBY_VERSION < '2.2.5'
+    gem 'beaker', '< 3'
```

Comment:
can you use the oneliner notation instead. More compact and still really readable.